bin/docker-dev-shell

```diff
@@ -160,6 +160,7 @@ docker run --rm -ti \
   -v "$(pwd)/bundler/Gemfile:$CODE_DIR/bundler/Gemfile" \
   -v "$(pwd)/bundler/dependabot-bundler.gemspec:$CODE_DIR/bundler/dependabot-bundler.gemspec" \
   -v "$(pwd)/bundler/lib:$CODE_DIR/bundler/lib" \
+  -v "$(pwd)/bundler/helpers:$CODE_DIR/bundler/helpers" \
   -v "$(pwd)/bundler/spec:$CODE_DIR/bundler/spec" \
   -v "$(pwd)/bundler/helpers:/opt/bundler" \
   -v "$(pwd)/bundler/helpers:/opt/bundler/helpers" \
```

Comment:
Question: is this `/opt/bundler/helpers` mount actually used? It seems eerily close to the `$CODE_DIR/bundler/helpers` mount we're adding.

I have no idea, just wondering if this was how this was _supposed_ to work, and could be cruft after this fix to how it _actually_ works. I would not be surprised if both are needed for difference cases (e.g. `rspec` vs `bin/dry-run.rb`).

This entire comment is idle curiosity and safe to ignore.

Reply:
I think both are needed for now as `rspec` loads stuff from the code dir, but might only be needed for native helper specs. We copy native helpers to `/opt/bundler` and run them from there when running dry-runs/tests. 

We might be able to simplify this with the bundler 2 work, planning to move the native helpers specs into the helper folder and run `rspec` from there.